datafusion/core/tests/parquet/arrow_statistics.rs

```diff
@@ -159,11 +161,14 @@ impl TestReader {
 
         // open the file & get the reader
         let file = file.reopen().unwrap();
-        ArrowReaderBuilder::try_new(file).unwrap()
+
+        // tell the reader to read the page index, if it exists
+        let options = ArrowReaderOptions::new().with_page_index(true);
+        ArrowReaderBuilder::try_new_with_options(file, options).unwrap()
     }
 }
 
-/// Defines a test case for statistics extraction
```

Comment:
This is kind of a wierd test change, but I was trying to setup a pattern where we didn't have to change all the tests at once